account_move_reconcile_helper/models/account_move_line.py

```diff
@@ -44,11 +44,7 @@ def _get_reconciled_lines(self, move_lines=None):
 
         for line in self:
             if line.full_reconcile_id:
-                partial_reconciles = line.full_reconcile_id.mapped(
-                    'partial_reconcile_ids')
-                matched_lines = (
-                    partial_reconciles.mapped('debit_move_id') |
-                    partial_reconciles.mapped('credit_move_id'))
+                matched_lines = line.full_reconcile_id.reconciled_line_ids
             elif line.credit > 0:
                 matched_lines = line.matched_debit_ids.mapped('debit_move_id')
```

Comment:
Last question: are you sure this test is correct? There is never a matched credit with a credit? eg when you partial reconcile 2 debits with 3 credits in one operation?